mixins/fluid.scss

```
@@ -0,0 +1,33 @@
+@use "sass:meta";
+@use "sass:string";
+@use "sass:math";
+
+///
+/// Returns a clamp function with calculated values, depending on the given params.
+///
+@function fluid-prop($min-vw, $max-vw, $min-value, $max-value) {
+	/* prettier-ignore */
+	@if meta.type-of($min-vw) == "number"
+		and not math.is-unitless($min-vw)
+		and meta.type-of($max-vw) == "number"
+		and not math.is-unitless($max-vw)
+		and meta.type-of($min-value) == "number"
+		and not math.is-unitless($min-value)
+		and meta.type-of($max-value) == "number"
+		and not math.is-unitless($max-value)
+		and math.unit($min-vw) == math.unit($max-vw)
+		and math.unit($min-vw) == math.unit($min-value)
+		and math.unit($min-vw) == math.unit($max-value)
```

```suggestion
	@if meta.type-of($min-vw) == "number"
		and meta.type-of($max-vw) == "number"
		and meta.type-of($min-value) == "number"
		and meta.type-of($max-value) == "number"
		and not math.is-unitless($min-vw)
		and math.unit($min-vw) == math.unit($max-vw)
		and math.unit($min-vw) == math.unit($min-value)
		and math.unit($min-vw) == math.unit($max-value)
```

Should be enough, right? Because the checks at the end should cover the `is-unitless()`?